packages/react-components/react-charts-preview/library/docs/Spec.md

```diff
@@ -0,0 +1,63 @@
+# @fluentui/react-charts-preview Spec
```

Comment:
This should be filled in or have a comment about when it will be filled in

Reply:
Updated this section with the relevant documentation.